tools/phpcs/phpcs.ruleset.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0"?>
-<ruleset name="WPP-Base">
+<ruleset name="WPP-Base" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
 	<description>Sniffs for WordPress plugins</description>
 
+	<rule ref="../../vendor/wpackagist-plugin/plugin-check/vendor/plugin-check/phpcs-sniffs/PluginCheck/ruleset.xml" />
```

Comment:
Why the double vendor? 🤔  Is this from the composer installer-paths config?

Reply:
Because it's installed in `vendor` by composer for this project, and then the plugin is distributed with its own `vendor` directory.